python/world/world.cpp

```diff
@@ -40,13 +40,16 @@ void python_world(py::module m) {
     .def("Execute", &World::Execute)
     .def("Observe", &World::Observe)
     .def("AddAgent", &World::AddAgent)
+    .def("RemoveAgentById", &World::RemoveAgentById)
     .def("AddObject", &World::AddObject)
     .def("GetParams", &World::GetParams)
     .def("UpdateAgentRTree", &World::UpdateAgentRTree)
     .def("ClearEvaluators", &World::ClearEvaluators)
     .def("SetMap", &World::SetMap)
     .def("AddEvaluator", &World::AddEvaluator)
-    .def("GetNearestAgents", &World::GetNearestAgents)
+    .def("AddLabels", &World::AddLabels)
+      .def_property_readonly("labels", &World::GetLabelFunctions)
```

Comment:
format

Reply:
This kind of problem happens often to me. I use the CLang Formatter with the format style file of Bark. When running the formatter on the file, it indents the lines by 4 spaces, as happened here. I have the impression that the formatter you use only has a 2 space indent, but the .clang-format file says 4 spaces for continuation.

https://github.com/bark-simulator/bark/blob/cc393384134188bf68d098a9efa486ff36069577/.clang-format#L52

Could you clarify this?

Reply:
I am currently working under macOS and do not have this problem.
Maybe @klemense1 could shed some light on this issue?

Reply:
Luis is right, the indentation here is not correct according to the clang file.